openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1206,3 +1214,88 @@ def validate_glusterfsd_memory_usage(self, size_limit):
                     "Failed memory used  of glusterfsd {} is greater than the"
                     " expected size {} for node/pod {}".format(
                         memory_used, size_limit, gluster_node))
+
+    def heketi_volumes_in_batch(
```

Comment:
Change function name to `create_heketi_volumes_in_batch` and parameter `kwargs` for passing heketi volume params.

Reply:
Done